all/projectAlcazar/include/timer.h

```diff
@@ -8,11 +8,295 @@
 #ifndef TIMER_H
 #define TIMER_H
```

Comment:
For the registers in timer that have bits which have different meaning base on oc mode you could define 2 structs inside the union one named bitFinction1 the other bitFunction2 where Function1 and Func2 are maybe some good abreviations of the 2 functions